FeedStoreChallenge/ManagedCache.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  ManagedCache.swift
+//  FeedStoreChallenge
+//
+//  Created by Joshua Bryson on 9/9/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import CoreData
+
+@objc(ManagedCache)
+internal class ManagedCache: NSManagedObject {
+	@NSManaged internal var timestamp: Date
+	@NSManaged internal var feed: NSOrderedSet
+}
+
+extension ManagedCache {
+	static func deletePreviousCache(in context: NSManagedObjectContext) throws {
+		try find(in: context).map(context.delete).map(context.save)
+	}
```

Comment:
Saving the context violates SRP. This method both deletes the object and saves the context.
Please move `context.save` outside of the function.
```suggestion
	static func deletePreviousCache(in context: NSManagedObjectContext) throws {
		try find(in: context).map(context.delete)
	}
```